src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -318,18 +317,17 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
       });
     }
   }, []);
-  /*
+
   useEffect(() => {
-    const handleContextmenu = (e) => {
-      e.preventDefault();
-      e.stopPropagation();
-      setIsOpen((prev) => !prev);
-    };
-    document.addEventListener('mousedown', handleContextmenu);
-    return function cleanup() {
-      document.removeEventListener('mousedown', handleContextmenu);
-    };
-  }, []);*/
+    newStorageService
+      .hasUploadedFiles()
+      .then(({ hasUploadedFiles }) => {
+        setHasAnyUploadedFile(hasUploadedFiles);
+      })
+      .catch((error) => {
+        errorService.reportError(error);
+      });
```

Comment:
Maybe it is not necessary to execute this call every time.
If I remember correctly, the onboarding is only shown on the day that the user has registered. 
So if it is no longer that day, it is not necessary to execute the call. 
I would also check if there are any other conditions to take into account in order not to make unnecessary calls to the backend.
While we are at it, maybe you can take the logic out of showing the onboarding to a hook.
If this last suggestion is too time consuming, there is no need to do it now

Reply:
Yep, I thought about extracting the logic to a custom hook, but I think we need this bug fixed ASAP 😄. Anyway, let me see what I can do with this.

on the other hand, I will add a check if the account has been created today.